test/pgw_proxy_SUITE.erl

```diff
@@ -18,6 +18,8 @@
 -include("ergw_pgw_test_lib.hrl").
 
 -define(TIMEOUT, 2000).
+-define(NO_OF_CLIENTS, 6). %% No of IP clients for multi session tests
```

Comment:
When I see NO I think of none, or no such or negations. If number is too long, how about num ?

Reply:
changed it to NUM_OF_CLIENTS.